.github/workflows/release.yml

```diff
@@ -99,14 +99,16 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Install and configure Poetry
-        uses: snok/install-poetry@v1
-        with:
-          version: 1.8.5
+        run: pipx install poetry
+      - name: Hotswap build backend for Poetry
+        # Maturin doesn't support building no-extension wheels, so we swap to Poetry for that
+        run: |
+          sed -i '/^\[build-system\]/,/^\[/{s/^requires = .*/requires = ["poetry-core>=2.0.0,<3.0.0"]/ s/^build-backend = .*/build-backend = "poetry.core.masonry.api"/}' pyproject.toml
       - name: Install dependencies
         run: poetry install --only main --only test --only typing --only build
       - name: Run poetry build
         run: poetry build
-      - name: Upload sdist
+      - name: Upload no-ext wheel
```

Comment:
What is in this wheel?

Reply:
Rust code has a pure python alternative available, see code paths [here](https://github.com/search?q=repo%3Apython-pendulum%2Fpendulum%20with_extensions&type=code). It's a fallback wheel.